pysolr.py

```diff
@@ -386,10 +386,12 @@ def _send_request(self, method, path='', body=None, headers=None, files=None):
                       url, method, log_body[:10], end_time - start_time)
 
         if int(resp.status_code) != 200:
-            error_message = self._extract_error(resp)
-            self.log.error(error_message, extra={'data': {'headers': resp.headers,
-                                                          'response': resp.content}})
-            raise SolrError(error_message)
+            error_message = "Solr responded with error (HTTP %s): %s"
+            params = (resp.status_code, self._extract_error(resp))
```

Comment:
Since this has only two values, maybe we should just pass them to log.error directly without using `*args` notation for clarity?